build.gradle.kts

```diff
@@ -12,15 +12,38 @@ plugins {
     id("org.liquibase.gradle") version Versions.liquibaseGradlePlugin
 }
 
+val profile = if(properties["profile"] == null) "dev" else properties["profile"]
+val propertyFileLines = File("save-backend/src/main/resources/application-$profile.properties").readLines()
+val secretsLines = File("secrets").readLines() // TODO: specify path to file
+
+val databaseUrl = getCredential(propertyFileLines, "database.url")
+var username: String
+var password: String
+
+if (profile == "prod") {
+    username = getCredential(secretsLines, "username")
+    password = getCredential(secretsLines, "password")
+} else {
+    username = getCredential(propertyFileLines, "database.username")
+    password = getCredential(propertyFileLines, "database.password")
+}
+
+fun getCredential(lines: List<String>, token: String): String =
```

Comment:
Use `Properties().load(...)` instead of manual parsing

Reply:
Fixed